### ISSUE TITLE
Print the jOrgChart

### ISSUE THREAD

**Issue by Nikolasgs** [wesnolte/jOrgChart#58]
Hello, 
I need to print the org. chart graphic, can I do that?

Regards.

**Comment by hudhaifa93** [comment]
am also having the save problem.. were u able to find a solution ?

**Comment by Nikolasgs** [comment]
Hi! I dont remember now... but I think I render an HTML page and print it.

**Comment by hudhaifa93** [comment]
can u send me the code how u did it

On 18 August 2015 at 18:39, Nicolás <notifications@github.com> wrote:

> Hi! I dont remember now... but I think I render an HTML page and print it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wesnolte/jOrgChart/issues/58#issuecomment-132200353>.
>



-- 
*Thanks & Regards,*
*Hudhaifa Yoosuf*

**Comment by Nikolasgs** [comment]
No, I´m sorry, I dont have it. It is from my previous job. But I think I created a new page (windows.open) with the chart in it, send it to the printer and close it. The user don´t see any new page... It´s a simple solution, very rustic.

**Comment by dabeng** [comment]
Hi @Nikolasgs @hudhaifa93 , you guys can make use of option "exportButton" of **[OrgChart](https://github.com/dabeng/OrgChart)** to accomplish printing task.

This is the **[demo](https://github.com/dabeng/OrgChart)**.

The following is screen-capture.
![screen-capture](https://cloud.githubusercontent.com/assets/5689075/15099001/53bde858-1510-11e6-8b90-bda5629c266e.gif)
Good luck! :blush:

**Comment by dabeng** [comment]
The mission of **[OrgChart](https://github.com/dabeng/OrgChart)** is to keep jOrgChart alive. :blush:
